grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -932,9 +932,9 @@ class UrlMappings {
             action = [GET: "read", POST: "update", PUT: "update", DELETE: "delete"]
         }
 
-        "/api/facilities/$facilityId/cycle-count-candidates" {
+        "/api/facilities/$facilityId/cycle-counts" {
```

Comment:
as explained quickly on the tech huddle - I didn't know whether to keep the `cycle-count-candidates` and create `cycle-counts` even though they would point to the same endpoint, or to have a common one. 
I'm up for suggestions.

Reply:
yeah it is a bit weird. My other comment suggested using just the one GET /cycle-counts/candidates API for all the tabs.

I get that it's a bit strange since for the other tabs like TO COUNT we could look at it like the primary resource we're fetching is cycle count (and we're joining candidate information to that) but since all tabs need essentially the same information, I think it's fine that they all use the GET candidates API